testing/tests/DevExpress.ui.widgets.scheduler/integration.recurringAppointments.tests.js

```diff
@@ -1472,3 +1472,36 @@ QUnit.test("Appointment has correct occurrences dates with interval > 1, custom
     assert.roughEqual(fourthPosition.top - firstPosition.top, eighthPosition.top - fourthPosition.top, 0.5, "Appointment's top are correct");
     assert.roughEqual(seventhPosition.top, eighthPosition.top, 0.5, "Appointment's occurrences after WKST are positioned correct on top");
 });
+
+QUnit.test("Recurrence appointment occurrences should have correct start date with timezone changing (T818393)", function(assert) {
+    this.createInstance({
+        views: ["day", "week", "workWeek", "month"],
+        currentView: "week",
+        startDayHour: 1,
+        firstDayOfWeek: 2,
+        height: 600,
+        dataSource: [{
+            text: "Recurrence",
+            startDate: new Date(2019, 2, 30, 2, 0),
+            startDateTimeZone: 'Europe/Copenhagen',
+            endDate: new Date(2019, 2, 30, 10, 0),
+            endDateTimeZone: 'Europe/Copenhagen',
+            recurrenceException: "",
+            recurrenceRule: "FREQ=DAILY"
+        }],
+        timeZone: 'Europe/Copenhagen',
```

Comment:
Strange thing: the `timeZone` option is not work and that test work always before I change time zone in my computer. Is it a bag?